cmd/operator-sdk/cli/cli.go

```diff
@@ -42,91 +30,71 @@ import (
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"sigs.k8s.io/kubebuilder/pkg/cli"
+	kbgov1 "sigs.k8s.io/kubebuilder/pkg/plugin/v1"
+	kbgov2 "sigs.k8s.io/kubebuilder/pkg/plugin/v2"
 )
 
-// GetCLIRoot is intended to creeate the base command structure for the OSDK for use in CLI and documentation
-func GetCLIRoot() *cobra.Command {
-	root := &cobra.Command{
-		Use:   "operator-sdk",
-		Short: "An SDK for building operators with ease",
-		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			if viper.GetBool(flags.VerboseOpt) {
-				if err := projutil.SetGoVerbose(); err != nil {
-					log.Fatalf("Could not set GOFLAGS: (%v)", err)
-				}
-				log.SetLevel(log.DebugLevel)
-				log.Debug("Debug logging is set")
-			}
-			if err := checkGoModulesForCmd(cmd); err != nil {
-				log.Fatal(err)
-			}
-		},
-	}
+var commands = []*cobra.Command{
+	// Once the KB CLI is made the default, add the "new" command as a way to
+	// scaffold the legacy project layout and mark "new" as deprecated
+	// new.NewCmd()
 
-	root.AddCommand(
-		add.NewCmd(),
-		alpha.NewCmd(),
-		build.NewCmd(),
-		bundle.NewCmd(),
-		cleanup.NewCmd(),
-		completion.NewCmd(),
-		execentrypoint.NewCmd(),
-		generate.NewCmd(),
-		migrate.NewCmd(),
-		new.NewCmd(),
-		olm.NewCmd(),
-		printdeps.NewCmd(),
-		run.NewCmd(),
-		scorecard.NewCmd(),
-		test.NewCmd(),
-		version.NewCmd(),
-	)
+	alpha.NewCmd(),
+	bundle.NewCmd(),
+	cleanup.NewCmd(),
+	completion.NewCmd(),
+	olm.NewCmd(),
+	run.NewCmd(),
```

Comment:
HI @estroz and @hasbro17, 
  
The go.mod scaffolded should not have the sdk modules? 
See its result:

```
 $ cat go.mod 
module example.com/m/v2

go 1.13

require (
	github.com/go-logr/logr v0.1.0
	github.com/onsi/ginkgo v1.11.0
	github.com/onsi/gomega v1.8.1
	k8s.io/apimachinery v0.17.2
	k8s.io/client-go v0.17.2
	sigs.k8s.io/controller-runtime v0.5.0
)


```

Reply:
The base go scaffold in kubebuilder does not use operator-sdk packages.

Reply:
@camilamacedo86 Right now nothing in the default scaffold is importing anything from the SDK libraries (e.g `pkg/log/zap` or `pkg/metrics`) so the operator-sdk wouldn't really show up in go.mod. Much like `sigs.k8s.io/kubebuilder` doesn't show up in kubebuilder projects.

This may change if we decide to override the default plugin scaffolds to change anything later on.

Reply:
I think would be nice we add our scaffolds and etc as add the sdk modules to allow the users to use it. But it is out of the scope of this PR.. so no worry. tks for the clarifications.